userService/src/main/java/com/userServiceCrudOps/springbootcrudeUserService/userServiceCrudApplication.java

```diff
@@ -0,0 +1,26 @@
+package com.userServiceCrudOps.springbootcrudeUserService;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import io.github.cdimascio.dotenv.Dotenv;
+
+@SpringBootApplication
+public class userServiceCrudApplication {
+
+    public static void main(String[] args) {
+        
+        Dotenv dotenv = Dotenv.load();
+        
+        
+        System.setProperty("SERVER_PORT", dotenv.get("SERVER_PORT"));
+        System.setProperty("MYSQL_URL", dotenv.get("MYSQL_URL"));
+        System.setProperty("MYSQL_USERNAME", dotenv.get("MYSQL_USERNAME"));
+		System.setProperty("MYSQL_PASSWORD", dotenv.get("MYSQL_PASSWORD"));
```

Comment:
Proper linting required. No extra spaces between two lines. You can use 1 extra space to distinguish different parts but avoid 2 or more spaces. Last System.setProperty() should be in same indentation as others.